Fix: code fix

hyunparrot opened the pull request:
------------------------------------------
Fix: Constants.kt changed enum class
Fix: Accordingly changed some code MyVideoAdapter.kt 
Rename: renamed few layout(snakeName)
longscaleshortsitem.xml => long_scale_shorts_item.xml

Constants.kt 클래스 enum Class로 변경,
그에 따른 MyVideoAdapter.kt도 Constants.kt변경에 맞게 코드 수정
몇몇 레이아웃 snakeName으로 수정

hyunparrot committed:
Fix: code fix
------------------------------------------
Fix: Constants.kt changed enum class
Fix: Accordingly changed some code MyVideoAdapter.kt
Rename: renamed few layout(snakeName)
longscaleshortsitem.xml => long_scale_shorts_item.xml

Constants.kt 클래스 enum Class로 변경,
그에 따른 MyVideoAdapter.kt도 Constants.kt변경에 맞게 코드 수정
몇몇 레이아웃 snakeName으로 수정

sunjoolee commented on app/src/main/java/com/example/myapplication/Constants.kt:
클래스 이름 오타 수정 & 코틀린 파일 이름 클래스 이름과 동일하게 바꿔주세요!

hyunparrot committed:
Rename: Rename Class Name
------------------------------------------
Rename: Constants.kt => YouTubeViewType.kt
Rename: enum Class YouTubeVieType => enum Class YouTubeViewType

sunjoolee reviewed (approved):
넵 좋습니다!!
